app/Resources/views/base.html.twig

```diff
@@ -7,20 +7,28 @@
         <title>{% block title %}{{ 'homepage.title' | trans({}, 'seo') }}{% endblock %}</title>
         <meta name="google" content="notranslate" />
         <meta name="description" content="{% block description %}{{ 'homepage.description' | trans({}, 'seo') }}{% endblock %}">
+        <link href="{{ asset('vendors/components/bootstrap/dist/css/bootstrap.min.css') }}" rel="stylesheet">
```

Comment:
I would have put bootstrap under the link of local files to upload them first, but otherwise good job Wafa! :)